qtop.py

```diff
@@ -1263,7 +1265,15 @@ def get_yaml_files(scheduler, filenames):
     return args[scheduler]
 
 
-def get_filenames_commands():
+def finalize_filepaths_schedulercommands():
+    """
+    returns a dictionary with contents of the form
+    {fn : (filepath, schedulercommand)}, e.g.
+    {'pbsnodes_file': ('/tmp/qtop_results_$USER/pbsnodes_a.txt', 'pbsnodes -a')}
+    if the -s switch (set sourcedir) has been invoked, or
+    {'pbsnodes_file': ('/tmp/qtop_results_$USER/pbsnodes_a<some_pid>.txt', 'pbsnodes -a')}
+    if ran without the -s switch.
+    """
```

Comment:
the following section (until line 1633) is heavily refactored, it might not make much sense from the PR view, but it reads a helluva lot better in the .py file!!